app/src/main/java/com/android/partagix/ui/screens/InventoryCreateOrEditItemScreen.kt

```diff
@@ -109,7 +117,25 @@ fun InventoryCreateOrEditItem(
               Box(
                   contentAlignment = Alignment.Center,
                   modifier = modifier.fillMaxHeight().fillMaxWidth(.4f).testTag("image")) {
-                    MainImagePicker()
+                    val image = File(uiImage.toString().ifEmpty { "default-image.jpg" })
+                    MainImagePicker(listOf(image.toUri())) { uri ->
+                      // TODO :  Save the image to a local file to its displayed correctly while
+                      // waiting for the upload
+                      /*
+                      val localFilePath = kotlin.io.path.createTempFile("temp-${i.id}", ".tmp").toFile()
+                      Missing : save the image to the local file (need a ContentResolver ?)
+                      uiImage = localFilePath
+                       */
```

Comment:
These 2 commented blocks of code contain half the work already done for the TODOs that are just over them, I think it is best to leave them as this will be useful in the future.

Reply:
Okay then you're right it's best to leave it